index.d.ts

```diff
@@ -64,6 +55,12 @@ interface Options {
   //   }
   // },
 
+  /**
+   * Add extra code for development and debugging purposes.
```

Comment:
I think this doesn't really add "extra code", but pretty prints the sourcemap

Reply:
For this plugin, yes, but it's forwarded to `svelte/compiler` :: https://svelte.dev/docs#svelte_compile